Dockerfile

```diff
@@ -36,7 +36,9 @@ ENV ALLOW_FREE_ASSETS_ONLY='false'
 # docker-entrypoint.sh configuration file variables
 ENV AQUARIUS_WORKERS='8'
 ENV EVENTS_ALLOW=''
-ENV RUN_EVENTS_MONITOR=''
+ENV RUN_EVENTS_MONITOR='1'
+ENV PROCESS_POOL_EVENTS='1'
+ENV PROCESS_PURGATORY='1'
```

Comment:
There is already a `UPDATE_ALL_PURGATORY` env var, can we just use that instead of introducing a second env var?